finder-app/finder.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
@ameyflash avoid committing the empty files `aeld-data` and `ameyflash1.txt` -add to .gitignore if needed.